recipes/conda-which/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "conda-which" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kelvinou01/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  path: ../
```

Comment:
You probably want to remove the `path` specification. This looks to be leftover from your local recipe in the upstream repo.